build-tools/src/testFixtures/groovy/org/elasticsearch/gradle/fixtures/AbstractGradleFuncTest.groovy

```diff
@@ -136,6 +143,35 @@ abstract class AbstractGradleFuncTest extends Specification {
         return jarFile;
     }
 
+    //todo maybe we could refer to ../libs/plugin-api compile it and create a jar?
+    File pluginApiJar(File jarFolder){
```

Comment:
Is this stuff used somewhere at all?

Reply:
Should be moved to the specific test and not live in the common abstract base class

Reply:
I created a utility class for these methods as they are also used in a NamedComponentScannerSpec